tests/Steam.Common.Tests/SteamIdTests.cs

```diff
@@ -0,0 +1,152 @@
+﻿using System;
+using Xunit;
+
+namespace Steam.Tests
+{
+    public class SteamIdTests
```

Comment:
I'd like to see more tests to check the correctness of equality operators and preferably also the properties to make sure we can't accidentally break these.

Reply:
I believe the "parts" tests test these well.